src/utils/formDataUtils.ts

```diff
@@ -110,13 +111,10 @@ const getPatronAgencyType = (agencyTypeParam?) => {
  * getLocationValue
  * Map the location value from the form field into the string value.
  */
-const getLocationValue = (location: string): string => {
-  const locationMap = {
-    nyc: "New York City (All five boroughs)",
-    nys: "New York State (Outside NYC)",
-    us: "United States (Visiting NYC)",
-  };
-  return locationMap[location] || "United States (Visiting NYC)";
+const getLocationValue = (location = "en", lang = "en"): string => {
```

Comment:
Just noticed: Should the parameter `location` default to "us" rather than "en"?

Reply:
Yes! whoops.